test/helper.js

```diff
@@ -98,7 +98,7 @@ module.exports.payloadMethod = function (method, t, isSetErrorHandler = false) {
       return
```

Comment:
this `return` will cause `t.teardown` not to be called. 
To be inline with most other test files, suggest we remove the if and the `return`.

Reply:
Another way to resolve this one is normalise calling `t.teardown` immediately after fastify is instantiated as stated in the test docs and not done in many places.
Kinda like go's `defer`.